content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,23 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.10 [2020-05-07]
+
+### Features
+
+- Add ability to delete a stack and all associated resources.
+- Enforce DNS name compliance on all `pkger` resources' metadata.name field.
```

Comment:
should fields be plural?

Reply:
Yes, probably.

Reply:
🤔 the field that has to be compliant is singular (`metadata.name` field), but is enforced for all resources. I'm not sure what the correct grammar is now that I'm on the spot 😬

Reply:
Thanks, guys - @jsteenb2 would this be accurate?:
Enforce DNS name compliance on the `metadata.name` field in all `pkger` resources.

Reply:
smashing, that is spot on 👏